src/qml/MessageBubble.qml

```diff
@@ -100,18 +100,16 @@ BorderImage {
             return theme.name === "Ubuntu.Components.Themes.SuruDark" ? "lightGreen" : "green"
         }
     }
+    backgroundColor: color
     property bool completed
     Component.onCompleted: completed = true
-    source: completed ? "assets/" + color + "_bubble.sci" : ""
```

Comment:
Since we're no longer using these files from `assets/` would it not also be good to remove `assets/*_bubble.{png,sci}` files in this PR? Also, will switching to `UbuntuShape` be slower, or faster, at all?

Reply:
Yes was thinking about that.
I don't any troubles regarding speed, but someone with long conversation should try